### PR TITLE
  added alumni page with SSR from bocacode-api

### PR DESCRIPTION
**Pull request by jonathans199**

### CODE REVIEW

File: components/Hero/Hero.module.scss [hunk 1]
```diff
@@ -69,20 +69,47 @@
...
+      rgba(255, 255, 255, 0.9) 0%,
+      rgba(255, 255, 255, 0.9) 0%,
+      transparent 70%
+    ),
+    url('/img/restaurants-bk.jpeg');
+}
+
+.hero-alumni {
+  @include hero-styles;
+  // background-position-y: 80%;
```
Comment (brunomaria71) [inline components/Hero/Hero.module.scss:107]: can remove commented out code

File: pages/alumni/AlumniCard.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  return (
+    <div style={selection}>
+      <div className={`${styles['team-card']}`}>
+        <img
+          className={`${styles['team-image']}`}
+          src={`https://bocacode.com/assets/images/candidates/${slug}.png`}
+          alt={name}
+        />
+        <div className="team-image-hover">
+          {/* <img
```
Comment (brunomaria71) [inline pages/alumni/AlumniCard.js:17]: are we saving this for later? if not we can delete, if not disregard

Comment (jonathans199) [reply pages/alumni/AlumniCard.js:17]: Yea this one is for the hover state, which we could enable later when we get the rest of the styles etc... 😎

File: pages/alumni/AlumniCard.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import styles from './Alumni.module.scss'
+import Link from 'next/link'
+
+function AlumniCard({ id, selectedCardId, image, name, link, about, linkedin, hired, slug }) {
+  const selection = selectedCardId ? { display: id === selectedCardId ? 'block' : 'none' } : {}
```
Comment (brunomaria71) [inline pages/alumni/AlumniCard.js:6]: loving the use of conditional class rendering

File: pages/alumni/AlumniCard.js (hunk 2 above)
Comment (brunomaria71) [reply pages/alumni/AlumniCard.js:17]: nice :D

**Review by brunomaria71 — APPROVED** [review]
fantastic work!!

**Pull request merged**